web/index.html

```diff
@@ -40,11 +40,12 @@ <h1><a href="/">Papyr</a></h1>
             <component v-bind:is="currentView">
                 <!-- component changes when vm.currentView changes! -->
             </component>
+
         </div>
     </div><!-- /container -->
 
     <script id="t-loading" type="text/x-template">
-        <div class="component">Loading...</div>
+        <div class="component">Loading... meanwhile type keywords such india, adams, etc. or categories such as crime, love, fiction to look for books</div>
     </script>
```

Comment:
The loading screen is only shown inbetween views. I would suggest removing the meanwhile... part and leave it as loading...